conbench/entities/run.py

```diff
@@ -135,7 +142,14 @@ def create(data) -> "Run":
         )
 
         run = Run(**data, commit_id=commit_id, hardware_id=hardware.id)
-        run.save()
+        try:
+            run.save()
+        except s.exc.IntegrityError as exc:
+            if "unique constraint" in str(exc) and "run_pkey" in str(exc):
+                raise EntityExists(
+                    f"conflict: a Run with ID {data['id']} already exists"
+                )
```

Comment:
I think you might need to still raise the original exception in an `else` clause here, just in case there's an `s.exc.IntegrityError` that isn't related to a `run_pkey` unique constraint

Reply:
Thank you for catching that; this was indeed absolutely my intention (the main reason for doing all this narrowing down is to re-raise all other exceptions). Will add a commit.